langchain-interpreter/src/langchain_interpreter/interfaces/webhook.py

```
@@ -0,0 +1,640 @@
+                    status_code=401,
+                    detail="Invalid signature",
+                )
+
+        try:
+            # Parse payload
+            payload = json.loads(body)
+        except json.JSONDecodeError:
+            raise HTTPException(
+                status_code=400,
+                detail="Invalid JSON payload",
+            )
```

Ruff (0.14.14)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)